internal/pkg/upgrade/upgrade.go

```diff
@@ -36,6 +38,16 @@ func Upgrade(continueDirectly bool) error {
 	}
 	log.Debugf("Dtm upgrade: work path is : %v", workDir)
 
+	// get dtm bin file path like `/usr/local/bin/dtm-linux-amd64`
+	binFileDir, err := os.Executable()
```

Comment:
Executable() function returns the path name for the executable that started the current process, So the first return value here is better named `binFilePath` instead of `binFileDir`. Or `dtmFilePath` is also a good var name here.

Reply:
Sincerely thanks for your help, I 've already change it into binFilePath.